packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -140,8 +140,16 @@ const props = defineProps<{
 const doc = ref<DocumentAsset | null>(null);
 const pdfLink = ref<string | null>(null);
 const view = ref(DocumentView.EXTRACTIONS);
-const viewOptions = ref([{ value: DocumentView.EXTRACTIONS, icon: 'pi pi-list' }]);
-const code = ref<string>();
+const viewOptions = computed(() => {
+	if (doc.value?.fileNames?.at(0)?.endsWith('.pdf')) {
+		return [extractionsOption, pdfOption];
+	}
+	return [extractionsOption, txtOption];
+});
+const extractionsOption = { value: DocumentView.EXTRACTIONS, icon: 'pi pi-list' };
+const pdfOption = { value: DocumentView.PDF, icon: 'pi pi-file-pdf' };
+const txtOption = { value: DocumentView.TXT, icon: 'pi pi-file' };
```

Comment:
Please add markdown support as well using the text-editor.

Reply:
vue quill doesn't natively handle markdown but there are some vue quill modules I can try.

Reply:
No worries, then we should display Mardown as simple text for now.